src/components/menus/GridMenu.svelte

```diff
@@ -9,8 +9,11 @@
   import PlanIcon from '@nasa-jpl/stellar/icons/plan.svg?component';
   import TableWithHeaderIcon from '@nasa-jpl/stellar/icons/table_with_header.svg?component';
   import VerticalCollapseIcon from '@nasa-jpl/stellar/icons/vertical_collapse_with_center_line.svg?component';
+  import Balloon from 'bootstrap-icons/icons/balloon.svg?component';
   import CodeSquareIcon from 'bootstrap-icons/icons/code-square.svg?component';
+  import Cookie from 'bootstrap-icons/icons/cookie.svg?component';
```

Comment:
Why a Cookie?

Reply:
It went with the event theme; this is one of the things we asked Lyle about design-wise!